build-package.sh

```diff
@@ -169,9 +169,25 @@ if [ "$(uname)" == "Darwin" ]; then
 fi
 
 # Generate jazzy docs (macOS)
-if [ "${docs}" == "true" ] && [ "$(uname)" == "Darwin" ]; then
-    sourceScript "${projectFolder}/Package-Builder/jazzy.sh"
+#if [ "${docs}" == "true" ] && [ "$(uname)" == "Darwin" ]; then
```

Comment:
Is this supposed to be commented out?

Reply:
It was the old method of just invoking the jazzy script without checks, has now been removed.